### PR TITLE
changes to service/micro framework

### PR DESCRIPTION
**Pull request by aricart**
- removed `api_url` use `metadata`
- removed `SCHEMA` verb, schemas can be reported via the endpoint metadata
- added `metadata` on the endpoints report (was missing from the ADR)

### CODE REVIEW

File: adr/ADR-32.md [hunk 1]
```diff
@@ -25,15 +25,15 @@ Service configuration relies on the following:
...
+- `metadata` - (optional) an object of strings holding free form metadata about
+  the deployed instance implemented consistently with
+  [Metadata for Stream and Consumer ADR-33](ADR-33.md).
 - `statsHandler` - an optional function that returns unknown data that can be
   serialized as JSON. The handler will be provided the endpoint for which it is
   building a `EndpointStats`
-- `endpoint` - an optional base endpoint configuration, consisting of valid NATS subject,
-  handler and optional request/response schema. The `name` of the base endpoint 
-  should be set to `default` by the client libraries.
+- `endpoint` - an optional base endpoint configuration, consisting of valid NATS
```
Comment (ripienaar) [inline adr/ADR-32.md:34]: we keep this?

File: adr/ADR-32.md [hunk 2]
```diff
@@ -25,15 +25,15 @@ Service configuration relies on the following:
...
+  subject, handler and optional request/response schema. The `name` of the base
```
Comment (ripienaar) [inline adr/ADR-32.md:35]: still mentions schema here

**Review by ripienaar — CHANGES_REQUESTED** [review]

File: adr/ADR-32.md (hunk 1 above)
Comment (aricart) [reply adr/ADR-32.md:34]: Nope - I already took it out.

File: adr/ADR-32.md (hunk 2 above)
Comment (aricart) [reply adr/ADR-32.md:35]: the word `schema` is about the `schema` for the PING response fields - let me remove that

**Review by ripienaar — APPROVED** [review]
LGTM

**Review by Jarema — APPROVED** [review]
LGTM!

**Pull request merged**